spec/helpers/application_helper_spec.rb

```diff
@@ -120,7 +120,8 @@
     end
 
     it "has the correct text in the a tag" do
-      expect(helper.collection_link(collection)).to have_text("#{collection.slug} (#{collection.articles.published.size} Part Series)")
+      expect(helper.collection_link(collection)).to
```

Comment:
`The expect syntax does not support operator matchers, so you must pass a matcher to`

I think maybe this multiline syntax doesn't work?

Reply:
Got it: turns out the `.to` needs to be on the next line!